TestWordle/UnitTest1.cs

```diff
@@ -0,0 +1,51 @@
+// Test cases for wordle game
+
+using Training;
+
+namespace TestWordle {
+   [TestClass]
+   public class TestWordle {
+
+      // Test the backspace key input

```

Comment:
```suggestion
      // Tests the backspace key input

```
Change such comments everywhere